ssh-askpass.plist

```diff
@@ -12,7 +12,8 @@
             <string>-pc</string>
             <string>#!/bin/sh
 launchctl setenv SSH_ASKPASS "${SSH_ASKPASS:=/usr/local/bin/ssh-askpass}"
-launchctl setenv DISPLAY "${DISPLAY:=0}" # only if not already set by Xquartz
+launchctl setenv SUDO_ASKPASS "${SUDO_ASKPASS:=/usr/local/bin/ssh-askpass}"
+launchctl list org.xquartz.startx >/dev/null || launchctl setenv DISPLAY "${DISPLAY:=ssh-askpass}" # only if not already set by Xquartz
```

Comment:
```
"${DISPLAY:=ssh-askpass}"
```

Is that really correct? Shouldn't it be `:=0`?

Reply:
Yes! 

If the display environment is not already configured, then we do not know what it is. Zero could be a correct setting or it might not be. If there are multiple users on the computer, then it is quite likely that zero will be correct sometimes and incorrect other times.

If there is one user on the computer who uses any sort of nested or forwarded X 11, then again it is likely that sometimes zero will be correct and sometimes not. ￼￼

Reply:
If the user doesn't have Xquartz installed at all, then it doesn't matter what `$DISPLAY` is set to as it would never be used. Our value is only used to trick `ssh-add` and `ssh-agent`, never actually used for X.

Reply:
(I'm currently knee-deep in trying to get a patch in to Xquartz so I'm doing the weird and I tripped over my own nose here...hence the patch...)